.github/workflows/integration-tests.yml

```diff
@@ -33,11 +35,15 @@ jobs:
         run: |
           sudo apt install libmagic1
 
-      - name: Build frontend and swagger ui
+      - name: Build swagger ui
         run: |
-          ./scripts/codex/build.frontend.sh
           ./scripts/swagger/build.sh
 
+      - name: Create apiKeys.js for frontend
+        run: |
+          cd _frontend.codex
+          cp src/constants/apiKeysTemplate.js src/constants/apiKeys.js
```

Comment:
@mmulich `cp src/constants/apiKeysTemplate.js src/constants/apiKeys.js` is still necessary.  I'm wondering if it's better to include this in https://github.com/WildMeOrg/houston/blob/main/.dockerfiles/dev-frontend/docker-entrypoint.sh ?

Reply:
If we move this lets do it is a subsequent PR, I'd like to get this in right away so tests pass.